tests/test_zooniverseCutouts.py

```diff
@@ -32,6 +32,34 @@
 import PIL
 from lsst.analysis.ap import zooniverseCutouts
 
+# Sky center chosen to test metadata annotations
+skyCenter = lsst.geom.SpherePoint(245.0, -45.0, lsst.geom.degrees)
```

Comment:
Anything special about this point? Just wondering.

Reply:
I just picked a 3-digit RA and negative Dec, to make sure that looked ok in the output.